floris/simulation/turbine/turbine.py

```diff
@@ -130,7 +130,7 @@ def power(
     """
     # TODO: Change the order of input arguments to be consistent with the other
     # utility functions - velocities first...
-    # Update to power calculation which replaces the fixed pP exponent with
+    # Update to power calculation which replaces the fixed cosine_loss_exponent_yaw exponent with
```

Comment:
Similar to in the operation models module, this might be more clear in plane english